.agent/skills/ss14-physics-system-core/SKILL.md

```
@@ -0,0 +1,276 @@
+
+This skill covers the architecture and mental model of PhysicsSystem :)
+For the full directory of the public API, see the separate skill `SS14 Physics System API`.
+
+## Mental model
+
+Physics in SS14 is convenient to think of as a pipeline of 6 layers:
+
+1. `PhysicsComponent`
+- Body state: `BodyType`, `CanCollide`, speed, damping, mass, sleep, etc.
+
+2. `FixturesComponent`
```

markdownlint-cli2 (0.22.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)